modules/configuration/js/configuration_helper.js

```diff
@@ -86,6 +86,9 @@ $(function () {
             error: function(xhr, desc, err) {
                 console.log(xhr);
                 console.log("Details: " + desc + "\nError:" + err);
+                var html = "<label>Internal Server Error or Duplicate Values</label>";
```

Comment:
We should remove the "Internal Server Error" and catch separate errors and display the message appropriately

@driusan ??

Reply:
yeah, there's already details going to the console if someone needs to figure out what happened. I think we can improve the UI in a different PR if necessary.. but errors should be exceptional (I hope?), not a normal occurance, so i'm not sure how often this code path will even be triggered.